controller/volume_controller.go

```diff
@@ -801,14 +801,14 @@ func (vc *VolumeController) ReconcileVolumeState(v *longhorn.Volume, es map[stri
 				failedUsableReplicas[r.Name] = r
 			}
 			if !dataExists {
-				logrus.Warnf("volume controller: cannot auto salvage volume %v: no data exists", v.Name)
+				vc.logger.Warnf("Cannot auto salvage volume %v: no data exists", v.Name)
 			} else {
 				salvaged := false
 				for _, r := range failedUsableReplicas {
 					if util.TimestampWithinLimit(lastFailedAt, r.Spec.FailedAt, AutoSalvageTimeLimit) {
 						r.Spec.FailedAt = ""
 						msg := fmt.Sprintf("Replica %v of volume %v has been automatically salvaged", r.Name, v.Name)
-						logrus.Warnf(msg)
+						vc.logger.Warnf(msg)
```

Comment:
use Warn, since you are no longer using a format string.

Reply:
Also replace the message since we will be having fields for `replica` and `volume`